tsconfig.json

```diff
@@ -20,12 +23,7 @@
     "noImplicitReturns": true,
     "noPropertyAccessFromIndexSignature": true,
     "noUnusedLocals": true,
-    "noUnusedParameters": true,
-
-    /* Module Resolution Options */
-    "esModuleInterop": true,
-    "moduleResolution": "node16",
-    "resolveJsonModule": true
```

Comment:
`--module node16` implies all three of these things (though I had to add the explicit `esModuleInterop` back; see the comment above)